Mathlib/SetTheory/Cardinal/Basic.lean

```diff
@@ -826,6 +824,16 @@ theorem mk_diff_add_mk {S T : Set α} (h : T ⊆ S) : #(S \ T : Set α) + #T = #
   refine (mk_union_of_disjoint <| ?_).symm.trans <| by rw [diff_union_of_subset h]
   exact disjoint_sdiff_self_left
 
+lemma diff_nonempty_of_mk_lt_mk {S T : Set α} (h : #S < #T) : (T \ S).Nonempty := by
+  suffices (T \ S).Nonempty by simpa
```

Comment:
This looks exactly like the lemma statement. Is there something else going on, or can this line be removed?

Reply:
Good point! I'll amend.